spec/system/search_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+describe 'Search', type: :system, js: true do
+  let(:search_page) { PageObjects::Pages::Search.new }
+  fab!(:topic) { Fabricate(:topic) }
+  fab!(:post) { Fabricate(:post, topic: topic, raw: "This is a test post in a test topic") }
+
+  describe "when using full page search on mobile" do
```

Comment:
:100: I can't do a reaction on a file, so I will do it in a comment instead :laughing: